project/plugins.sbt

```diff
@@ -24,3 +25,5 @@ addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.10")
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.8.1")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.8.5")
+
+addSbtPlugin("org.jetbrains.teamcity.plugins" % "sbt-teamcity-logger" % "0.3.0")
```

Comment:
Added b200be6c269c401a8994b47c5315a0d3c91d496e to workaround this issue. Commented on the JetBrains ticket to see if we can fix it properly in the future. This will do for now I think though.